### PR TITLE
(#817) Remove QDE Docs

### LINKED ISSUE #817: Remove Deprecated QDE Docs
## What New Or Updated Would You Like To See?
The time has come to remove the QDE enviornment section from our documentation.

## Why Is It Needed?
We have added a deprecation notice to the current documentation for several months. Its time to fully take down the docs from the page entirely.

## Additional Context?
N/A

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Remove QDE documentation. Also removed any old xrefs between docs that used this outdated documentation.

## Motivation and Context
QDE is a deprecated solution. Thus the docs should be removed.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* ~[ ] Minor documentation fix (typos etc.).~
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* ~[ ] New documentation page added.~
* ~[ ] The change I have made should have a video added, and I have raised an issue for this.~
    * ~Issue #~

## Change Checklist

* [X] Requires a change to menu structure (top or left hand side)/
* [X] Menu structure has been updated

## Related Issue

Fixes #817

### CODE REVIEW

**Comment by pauby** [comment]
@gep13 I've added you as a reviewer to this.

**Comment by gep13** [comment]
@pauby the changes in this PR look good to me, but I do think we are missing something.

For example, what do we want to happen if someone has an existing link to the QDE environment page, for example, this link:

https://docs.chocolatey.org/en-us/c4b-environments/quick-deployment/

Do we want this to redirect through to the QSG home instead?  If so, we can put this in place using a `RedirectFrom` on that page.  If we go down this route, I would suggest that we should do it for each page that is being deleted.

Thoughts?

**Comment by steviecoaster** [comment]
> @pauby the changes in this PR look good to me, but I do think we are missing something.
> 
> For example, what do we want to happen if someone has an existing link to the QDE environment page, for example, this link:
> 
> https://docs.chocolatey.org/en-us/c4b-environments/quick-deployment/
> 
> Do we want this to redirect through to the QSG home instead? If so, we can put this in place using a `RedirectFrom` on that page. If we go down this route, I would suggest that we should do it for each page that is being deleted.
> 
> Thoughts?

I agree with Gary, we should redirect  each deleted page to the top-level page for the Quickstart Environment guide.

**Comment by gep13** [comment]
I actually had a chat with @pauby about this earlier today, but didn't get a chance to reply here again...

What we discussed was to create a new landing page, that doesn't exist in the navigation menu, that all the deleted links point to. That page would hold words about QDE going away, and a link to the root of QSG. That way, anyone navigating to a QDE won't be completely confused as to why they got to QSG.

Thoughts?

**Comment by gep13** [comment]
@ryanrichter94 @steviecoaster let me know your thoughts on this [comment](https://github.com/chocolatey/docs/pull/828#issuecomment-1726192951). Thinking about it more, I really feel that we need some form of landing page (hidden from the navigation menu), to capture anyone still coming to these pages.  Let me know if you have any questions about setting it up.

**Comment by ryanrichter94** [comment]
After discussing internally we will create an unlisted landing page directing all old QDE documentation to the new landing page. The landing page will discuss why the QDE solution has been deprecated as well as link off to Quick Start Guide and it's documentation that can be used for things such as SSL certificate renewal.

File: input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+Order: 10
+xref: qde-deprecation
+Title: Quick Deploy Environment Deprecation Notice
+Description: Landing page for old QDE links
+ShowInNavbar: false
+ShowInSidebar: false
+RedirectFrom:
+en-us/c4b-environments/quick-deployment/release-notes
```
Comment (steviecoaster) [inline input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md:9]: Your YAML is incorrect here, I think this needs 2 spaces in front of each one.

**Review by steviecoaster — CHANGES_REQUESTED** [review]

**Review by steviecoaster — CHANGES_REQUESTED** [review]

File: input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The same [nexus upgrade documentation](xref:upgrade-nexus) works for Quick Start Guide and Quick Deploy Environment solutions.
+
+### How Do I Upgrade Chocolatey Central Management(CCM)?
+
+Here is a link to our [Chocolatey Central Management Upgrade Documentation](xref:ccm-upgrade).
+
+### How Do I Upgrade Jenkins?
+
+New versions of the [Jenkins package](https://community.chocolatey.org/packages/jenkins) are being continuously published to the Chocolatey Community Repository.
```
[the remark this answers is not in the document]
Comment (ryanrichter94) [reply input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md:47]: Added Jenkins Upgrade doc and linked it to this section.

File: input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The Chocolatey For Business Azure Environment was later developed as our replacement for the Quick Deployment Environment Hyper-V to Azure offering. The Azure Environment is a completely cloud native solution offered through the [Microsoft Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/chocolateysoftwareinc1605695330527.c4b_azure_qde). It stands up the same components of the previous Quick Deploy Environment and current Quick Start Guide solutions. Just using Azure cloud tooling for the host VM and networking.
+
+## I have a Quick Deploy Environment Solution in place. What do I do now?
+
+The beauty is while we don't support the Quick Deploy Environment as a new deployment option. If already deployed, you have the same components (Sonatype Nexus, Jenkins, and Chocolatey Central Management) as our current environment solutions. You simply need to keep them up to date going forward.
```
[the remark this answers is not in the document]
Comment (ryanrichter94) [reply input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md]: Fixed wording.

File: input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Our [certificate renewal documentation](xref:quick-start-guide-cert-renewal) for Quick Start Guide works just fine on the older Quick Deploy Environment.
+
+### How Do I Upgrade My Nexus Instance?
+
+The same [nexus upgrade documentation](xref:upgrade-nexus) works for Quick Start Guide and Quick Deploy Environment solutions.
+
+### How Do I Upgrade Chocolatey Central Management(CCM)?
+
+Here is a link to our [Chocolatey Central Management Upgrade Documentation](xref:ccm-upgrade).
```
[the remark this answers is not in the document]
Comment (ryanrichter94) [reply input/en-us/c4b-environments/quick-start-environment/qde-deprecation.md]: Fixed wording.

**Review by steviecoaster — APPROVED** [review]

**Pull request merged**